packages/liferay-npm-scripts/src/scripts/test.js

```diff
@@ -23,7 +23,7 @@ const JEST_CONFIG = getMergedConfig('jest');
 /**
  * Main script that runs `jest` with a merged config
  */
-module.exports = function(arrArgs) {
+module.exports = function(arrArgs = []) {
```

Comment:
Note that I added this to make manual QA a bit easier. I noticed that I couldn't just do this in the console due to the expected array argument:

```
require('./src/scripts/test')()
```